packages/web-components/src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -11,7 +11,38 @@ import { consoleDevError } from '../../utils/utils';
   shadow: true,
 })
 export class VaOnThisPage {
+  /**
+   * The event used to track usage of the component. This is emitted when the
+   * user clicks on a link and enableAnalytics is true.
+   */
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  /**
+   * Whether or not an analytics event will be fired
+   */
+  @Prop() enableAnalytics?: boolean;
```

Comment:
I think we should get confirmation from someone (maybe Brian? I'm not sure) that analytics tracking for this component should be opt in rather than having it be the default, possibly with an option to opt out.

Reply:
Brian stated that analytics can be enabled by default unless:


- [ ] Any chance of pii / phi. So things like form input components.
- [ ] We see a potential for too many events. Things like the progress bar, which can be coded to send in an event every time the % changes.  No pii there, but we made it opt in.